pages.nl/common/ar.md

```diff
@@ -0,0 +1,25 @@
+# ar
+
+> Maken, aanpassen en uitpakken van Unix archieven. Typisch gebruikt voor statische bibliotheken (`.a`) en Debian pakketten (`.deb`).
+> Bekijk ook: `tar`.
+> Meer informatie: <https://manned.org/ar>.
+
+- Pak alles uit van een archief:
+
+`ar x {{pad/naar/bestand.a}}`
+
+- [t]oon inhoud van een specifiek archief:
```

Comment:
```suggestion
- [T]oon inhoud van een specifiek archief:
```

Not sure if this needs to be a capital letter?

Reply:
Since it's a mnemonic we can skip capitalising it here.